include/unicore-mx/usbd/usbd.h

```diff
@@ -643,6 +655,22 @@ uint8_t usbd_get_address(usbd_device *dev);
  */
 usbd_speed usbd_get_speed(usbd_device *dev);
 
+
+/**
+ * Get status of connected cable.
+ * @param[in] dev USB Device
+ * @return true - cable connected, and Vbus active
+ */
+bool usbd_is_vbus(usbd_device *dev);
+
+/**
+ * Controls power state of usb-core and PHY
+ * @param[in] dev USB Device
+ * @param[in] onoff  - true turn on device in action, and power PHY
+ *                     false disable PHY and stops usb-core
+ */
+void usbd_enable(usbd_device *dev, bool onoff);
+
```

Comment:
There is already a function, see "void usbd_disconnect(usbd_device *dev, bool disconnected);".